frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -328,6 +337,7 @@ export const funnelLogic = kea<funnelLogicType>({
     selectors: ({ props, selectors }) => ({
         isLoading: [(s) => [s.rawResultsLoading], (rawResultsLoading) => rawResultsLoading],
         results: [(s) => [s.rawResults], (rawResults) => rawResults.results],
+        resultsLoading: [(s) => [s.rawResultsLoading], (rawResultsLoading) => rawResultsLoading],
```

Comment:
Context on this change? It's the same selector as `isLoading`

Reply:
We need this for `resultsLoading` in `DashboardItem`. All logics except for `funnelLogic` had this `resultsLoading` selector being called in our dashboards